index.html

```diff
@@ -1,53 +1,53 @@
-				<a href="#" id="tweets-more" class="more-link hide">More&hellip;</a>
-			</div>
-			<div id="results-container" class="hide">
-				<div id="results-nada" class="hide">No results found. Try another search?</div>
-				<ul id="results-list" class="tweets-list media-list"></ul>
-				<a href="#" id="results-more" class="more-link hide">More&hellip;</a>
-			</div>
-		</div>
-	</div>
-	<div id="lightbox-cover"></div>
-	<div id="lightbox"></div>
-	<script src="config.js"></script>
-	<script src="assets/js/scripts.min.js"></script>
-	<script src="assets/js/app.js"></script>
-</body>
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Tweets</title>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+<meta name="format-detection" content="telephone=no">
+<link rel="shortcut icon" href="favicon.ico">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
+<link rel="stylesheet" href="assets/css/app.css">
+<body id="tweets-app">
+	<div class="tweets-container">
+		<header>
+			<h1 id="heading">Tweets</h1>
+			<p id="desc">An off-site archive of tweets from Twitter.</p>
+			<a href="#" id="home-link" class="header-link header-link-left hide" title="Go to home"><i class="fa fa-home"></i></a>
+			<a href="#" id="github-link" target="_blank" class="header-link header-link-right hide" title="Go to project on GitHub"><i class="fa fa-github"></i></a>
+		</header>
+		<div class="well-box">
+			<form id="search-form" class="hide">
+				<i class="fa fa-search"></i>
+				<input type="search" class="form-control search-query" placeholder="Search tweets" autocapitalize="off" autocorrect="off">
+			</form>
+			<h2>
+				<a href="#" class="toggle-tweets-nav">
+					<i class="fa fa-chevron-circle-down"></i>
+					<span class="heading-text"></span>
+				</a>
+			</h2>
+			<div id="tweets-nav" class="hidden">
+				<ul class="list-unstyled">
+					<li class="route-index"><a href="#"><span class="m">Recent tweets</span> <span class="n total-tweets"></span></a></li>
+				</ul>
+				<ul class="list-unstyled months-list">
+				</ul>
+			</div>
+			<div id="tweets-container" class="hide">
+				<ul id="tweets-list" class="tweets-list media-list"></ul>
+				<a href="#" id="tweets-more" class="more-link hide">More&hellip;</a>
+			</div>
+			<div id="results-container" class="hide">
+				<div id="results-nada" class="hide">No results found. Try another search?</div>
+				<ul id="results-list" class="tweets-list media-list"></ul>
+				<a href="#" id="results-more" class="more-link hide">More&hellip;</a>
+			</div>
+		</div>
+	</div>
+	<div id="lightbox-cover"></div>
+	<div id="lightbox"></div>
+	<script src="config.js"></script>
+	<script src="assets/js/scripts.min.js"></script>
+	<script src="assets/js/app.js"></script>
+</body>
```

Comment:
:open_mouth: Oh, didn't know that. Thought Atom editor would auto-fix it for me :confused: